packages/manager/modules/billing/src/autoRenew/autorenew.controller.js

```diff
@@ -198,6 +207,13 @@ export default class AutorenewCtrl {
   }
 
   onCriteriaChange($criteria) {
+    const newCriteria = $criteria.find(
```

Comment:
Once you found (or not) a new criteria, you need to update `this.currentCriteria` otherwise you'll always find the same changes in the list of criterion

Reply:
When onListParamChanges is called, the page reloads and onInit is executed with the new values cloned into currentCriteria.

Reply:
Indeed